### DISCUSSION TITLE
How can I deactivate the highlighting of problems in VScode?

### DISCUSSION THREAD

**Discussion by maxbruening** [James-Yu/LaTeX-Workshop#4055] [Q&A]
When I am drafting in VScode using the latex workshop extension, I am often not interested in seeing the problems of the current tex document highlighted in editor. For example, most of the time I do not need to see all my undefined citations. Is there a way to toggle these on and off?

**Comment by jlelong** [comment]
You have to deactivate the LaTeX linter. See https://github.com/James-Yu/LaTeX-Workshop/wiki/Linters
